tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/FmiIntegrationTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.Integration.Infrastructure;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.Identity.Test.Unit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+
+namespace Microsoft.Identity.Test.Integration.NetCore.HeadlessTests
+{
+    [TestClass]
+    public class FmiIntegrationTests
+    {
+        private const string _fmiAppUrn = "urn:microsoft:identity:fmi";
+        private const string _fmiRmaClientId = "4df2cbbb-8612-49c1-87c8-f334d6d065ad";
+        private const string _fmiNonRmaClientId = "";
+        private const string _fmiNonRmaClientIdInpersonation = "";
+        private const string _fmiTenantId = "f645ad92-e38d-4d1a-b510-d1b09a74a8ca";
+        private const string _fmiAuthority = "https://login.microsoftonline.com/" + _fmiTenantId;
+        private const string _fmiExchangeScope = "api://AzureFMITokenExchange/.default";
+        private const string _fmiExchangeScopeAsGuid = "022907d3-0f1b-48f7-badc-1ba6abab6d66/.default";
```

Comment:
What is this? It is not part of the spec? Is it some internal detail of how eSTS represents this scope?